js/app/Visualization/UI/CartoDBInfoWindow.js

```diff
@@ -0,0 +1,86 @@
+define([
```

Comment:
Why is this class in app/Visualization/UI ? I think it should live under app/Data somewhere as it doesn't actually build any UI, it fetches data from a CartoDB server (even if it's HTML data)...